src/Drupal/Generators/entity/hook_bundle_info.php.twig

```diff
@@ -0,0 +1,10 @@
+/**
+* Implements hook_entity_bundle_info_alter().
+*/
+function drush_empty_module_entity_bundle_info_alter(array &$bundles): void {
+{% for id in entity_type_ids %}
+  {% for bundle in infos[id]|keys %}
+    $bundles['{{ id }}']['{{ bundle }}']['class'] = \Drupal\{{ machine_name }}\Bundle\{{ id }}\{{ bundle|camelize }}::class;
```

Comment:
The default PHPCS lints will want this moved into a use statement. Given that it can also automatically fix the lint, I'd say no change is warranted here.

Reply:
That's supposed to be updated manually. So far generators can only append or replace files. Importing `use` statement would require parsing PHP code which is a way more complex.